src/ROUTES.ts

```diff
@@ -2390,6 +2390,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/accounting/sage-intacct/advanced/payment-account',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/accounting/sage-intacct/advanced/payment-account` as const,
     },
+    ADD_UNREPORTED_EXPENSE: {
+        route: 'search/r/:reportID/AddUnreportedExpense',
+        getRoute: (reportID: string | undefined) => `search/r/${reportID}/AddUnreportedExpense` as const,
```

Comment:
Let's follow our convention for route names here
```suggestion
        getRoute: (reportID: string | undefined) => `search/r/${reportID}/add-unreported-expense` as const,
```

Reply:
fixed